pkg/storage/mysql/mysql.go

```diff
@@ -187,9 +188,10 @@ func (m *MySQL) ReadUserTuple(ctx context.Context, store string, tupleKey *openf
 	objectType, objectID := tupleUtils.SplitObject(tupleKey.GetObject())
 	userType := tupleUtils.GetUserTypeFromUser(tupleKey.GetUser())
 
+	var conditionContext *[]byte
```

Comment:
Any particular reason for `*[]byte` here? The default value for a slice is nil, so that seems to cover the case. I tested out `[]byte` on my end and it works just as good.

Reply:
ah yeah good point 👍